index.js

```diff
@@ -10,8 +10,13 @@ const OK = chalk.green('Downloaded:');
 const FROM = chalk.gray('from');
 
 function download(repo, logger) {
-    utils.downloadRepo(repo).then((files) => {
-        files.forEach(file => logger.info(OK, `${file} ${FROM} ${repo}`));
+    utils.downloadRepo(repo).then(({ files, dependencies }) => {
+        if (files && files.length) {
+            files.forEach(file => logger.info(OK, `${file} ${FROM} ${repo}`));
+        }
+        if (dependencies && dependencies.length) {
+            dependencies.forEach(dep => logger.info(OK, `${dep} ${FROM} ${repo}`));
```

Comment:
I feel like it might be more useful to always show `<file> from <repo>`? That way user can see everything added to their project.

This would mean passing everything from `downloadRepo` thru to the same `then/catch` handlers.